docs/sample_config.yaml

```diff
@@ -1973,3 +1973,20 @@ opentracing:
     #
     #  logging:
     #    false
+
+
+## Cache Configuration ##
+
+# Caching can be configured through the following options.
+#
+caches:
+   # Controls the global cache factor. This overrides the "SYNAPSE_CACHE_FACTOR"
+   # environment variable.
```

Comment:
please can you explain what a global cache factor is, and document the default setting?

Reply:
I've attempted to do so.